EIPS/eip-3690.md

```diff
@@ -0,0 +1,266 @@
+---
+eip: 3690
+title: EOF - JUMPDEST Table
+author: Alex Beregszaszi (@axic), Paweł Bylica (@chfast), Andrei Maiboroda (@gumb0)
+discussions-to: https://ethereum-magicians.org/t/evm-object-format-eof/5727
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-06-23
+requires: 3540, 3670
+---
+
+## Abstract
+
+Introduce a section in the EOF format ([EIP-3540](./eip-3540.md)) for storing the list of `JUMPDEST`s, validate the correctness of this list at the time of contract creation, and remove the need for `JUMPDEST`-analysis at execution time. The `JUMPDEST` instruction is not needed anymore and becomes invalid.
```

Comment:
May want to make it clear in the abstract that the JUMPDEST instruction is invalid in contracts of the new type, but they will remain valid in contracts of the old type.

Reply:
Recommend mentioning what the prefix for this new type of contract is in the abstract.

Reply:
> Recommend mentioning what the prefix for this new type of contract is in the abstract.

We don't have that nailed down yet, but will rework this at the same time we rework the other EIPs (soon).

Reply:
> May want to make it clear in the abstract that the JUMPDEST instruction is invalid in contracts of the new type, but they will remain valid in contracts of the old type.

Added this.